keyboards/split_ergo/info.json

```
@@ -0,0 +1,15 @@
+{
+  "url": "",
+  "maintainer": "qmk",
+  "processor": "atmega32u4",
+  "bootloader": "caterina",
```

This can be changed to: 
```suggestion
  "development_board": "promicro",
```